Froliks.Calculator/Calculator/Program.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System.Text.RegularExpressions;
+
+namespace CalculatorProgram;
+
+public class Calculator
```

Comment:
Probably a nit, but maybe you can name this class `Program` as in the tutorial as it does seem confusing that a Calculator class is using another Calculator class.  Lmk what you think. 😄

Reply:
Yes, it is. I've changed the name to Program)